pydocx/export/image_resize.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+
+import base64
+from StringIO import StringIO
+from urlparse import urlparse
+import posixpath
+
+import requests
```

Comment:
This needs to be added to [install_requires](https://github.com/CenterForOpenScience/pydocx/blob/master/setup.py#L54).